File: 10.2D/Example code/#--CSharp/ShapeCSharp/Rectangle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ShapeCSharp
{
    public class Rectangle : Shape
    {
        private float _width;
        private float _height;
        public Rectangle(float x, float y, float width, float height) : base(x,y)
        {
            _width = width;
            _height = height;
        }

        public float Width { get => _width; set => _width = value; }
        public float Height { get => _height; set => _height = value; }

        public override void Draw(float x, float y)
        {
            //Draw Method Here...
        }
    }
}

File: 10.2D/Example code/Shape/Rectangle.java
public class Rectangle extends Shape {
    // private field
    private float _width;
    private float _height;

    // constructor
    public Rectangle(float x, float y, float width, float height) 
    {
        super(x,y);
        _width = width;
        _height = height;
    }

    public float get_height() {
        return _height;
    }

    public void set_height(float _height) {
        this._height = _height;
    }

    // property
    public float get_width() {
        return _width;
    }

    public void set_width(float _width) {
        this._width = _width;
    }

    // methods
    @Override
    public void Draw(float x, float y)
    {
        //draw function here
    }
}